build.gradle

```diff
@@ -14,17 +14,28 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+plugins {
+    id 'scala'
+    id 'java'
+    id 'idea'
+    id 'eclipse'
+    id 'cz.alenkacz.gradle.scalafmt' version "${scalamftVersion}"
+}
 
-buildscript {
+subprojects {
+    gradle.ext.scalafmt.configFilePath = gradle.ext.scalafmt.config
+}
+allprojects {
```

Comment:
```suggestion

allprojects {
```

Reply:
Here I have applied allprojects because there also sub-projects that need these settings.